app/assets/stylesheets/application.scss

```diff
@@ -36,66 +55,65 @@ p {
 #description {
   font-size: 175%;
   margin: 50px 1em 50px 1em;
-  padding: 2em 5em 2em 5em;
+  padding: 4em 5em 2em 5em;
 }
-.signup, .signup a {
+.sign-up-nav, .sign-up-nav a {
   color: white;
-  background: #cd2027;
-  border: 2px solid #cd2027;
+  background: #83090c;
+  border: 2px solid #83090c;
   -webkit-transition: all 0.6s;
   -moz-transition: all 0.6s;
   transition: all 0.6s;
 }
-.signup:hover, .signup a:hover {
-  color: #cd2027;
+.sign-up-nav:hover, .sign-up-nav a:hover {
+  color: #83090c;
   background: white;
-  border: 2px solid #cd2027;
+  border: 2px solid #83090c;
 }
-
 #signup-btn {
   height: 200px;
 }
 .btn-signup, .btn-signup:visited {
   text-decoration: none;
-  color: white;
-  background-color: #cd2027;
+  color: #ffffff;
+  background-color: #550000;
   font-size: 3em;
   height: 100px;
   width: 300px;
   border: 2px solid white;
+  padding: .375em;
   text-transform: uppercase;
   -webkit-transition: all 0.6s;
   -moz-transition: all 0.6s;
   transition: all 0.6s;
 }
-
 .btn-signup a {
-  color: white;
+  color: #550000;
   -webkit-transition: all 0.6s;
   -moz-transition: all 0.6s;
   transition: all 0.6s;
 }
 .btn-signup:hover {
-  border: 2px solid #cd2027;
-  background-color: white;
+  border: 2px solid #83090c;
+  background-color: #89ccbe;
+  color: #83090c;  
 }
 .btn-signup:hover a{
   text-decoration: none;
-  color: #cd2027;
+  color: #550000;
 }
-#person {
-  background-color: rgba(999, 999, 999, .4);
-  height: 100px;
-  width: 100px;
-  border: 2px solid #cd2027;
-  margin: 2em auto 2em 6em;
-  padding: 2em;
+.person {
+ margin-left: 3.5em;
+ padding: 1em;
 }
-#person-amount {
-  height: 20px;
-  border: 2px solid #cd2027;
-  margin-top: 30px;
-  max-width: 600px;
+.other-links {
+	margin-top: 10px;
+	margin-left: 60px;
+}
+.other-links a{
```

Comment:
Merge rule `.other-links a` with rule on line 109<br>Opening curly brace `{` should be preceded by one space